### PR TITLE
Release notes for oct 22 2023

### PR DESCRIPTION
**Pull request by abubinski**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-10-22
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* Kubernetes 1.25 is being deprecated end of January 2024 and support will transition to our [platform support policy](https://learn.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli#platform-support-policy).
```
Comment (CocoWang-wql) [inline CHANGELOG.md:9]: at the end of January 2024

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+* No new clusters can be created with [Azure AD Integration (legacy)](https://learn.microsoft.com/azure/aks/azure-ad-integration-cli). Existing AKS clusters with Azure Active Directory integration will keep working. All Azure AD Integration (legacy) AKS clusters will be migrated to [AKS-managed Azure AD](https://learn.microsoft.com/azure/aks/managed-azure-ad) automatically starting from 1st Dec. 2023. We recommend updating your cluster with AKS-managed Azure AD before 1 Dec 2023. This way you can manage the API server downtime during non-business hours.
+* [CVE-2023-29332](https://github.com/Azure/AKS/issues/3904) Microsoft Azure Kubernetes Service Elevation of Privilege Vulnerability. An attacker who successfully exploited this vulnerability could gain Cluster Administrator privileges. Please update your AKS VHD to at least VHD version 230801 as mentioned in the issue
```
Comment (CocoWang-wql) [inline CHANGELOG.md:11]: Add . at the end of sentence.

Shall we say 'please' in release notes? @palma21

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,35 @@
...
+
+### Release notes 
+* Features
+  * Bumped cloud-controller-manager image to v1.28.2, v1.27.10, v1.26.16 and v1.25.20. [Release notes](https://cloud-provider-azure.sigs.k8s.io/blog/).
```
Comment (CocoWang-wql) [inline CHANGELOG.md:15]: Pls move it to 'components update'

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,35 @@
...
+  * Added RBAC permissions to CNS for handling swiftv2 CRD resources("multitenantpodnetworkconfigs", "podnetworkinstances" and "podnetworks"). Note that swiftv2 flow is still under development and is not done yet. This change was done to unblock swiftv2 development.
```
Comment (CocoWang-wql) [inline CHANGELOG.md:16]: We need to add the links for line 16-21. BTW, are they all features? I think some of them are component updates and some are behavior changes~

File: CHANGELOG.md (hunk 3 above)
Comment (abubinski) [reply CHANGELOG.md]: fixed, thank you!

File: CHANGELOG.md (hunk 2 above)
Comment (abubinski) [reply CHANGELOG.md]: this announcement has been removed because it's not necessary

File: CHANGELOG.md (hunk 1 above)
Comment (abubinski) [reply CHANGELOG.md]: fixed, thank you!

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,33 @@
...
+* No new clusters can be created with [Azure AD Integration (legacy)](https://learn.microsoft.com/azure/aks/azure-ad-integration-cli). Existing AKS clusters with Azure Active Directory integration will keep working. All Azure AD Integration (legacy) AKS clusters will be migrated to [AKS-managed Azure AD](https://learn.microsoft.com/azure/aks/managed-azure-ad) automatically starting from 1st Dec. 2023. We recommend updating your cluster with AKS-managed Azure AD before 1 Dec 2023. This way you can manage the API server downtime during non-business hours.
+
+### Release notes 
+* Features
+  * Added RBAC permissions to CNS for handling swiftv2 CRD resources("multitenantpodnetworkconfigs", "podnetworkinstances" and "podnetworks"). Note that swiftv2 flow is still under development and is not done yet. This change was done to unblock swiftv2 development.
+  * Omsagent addon can now be interacted with using helm commands. The default value is set to true for omsagent addon v2 migration. 
+  * The default Fleet agent is now set to the new version (v1beta1), supporting the latest release in [Fleet](https://github.com/Azure/fleet).
+  * Bumped dropgz to v0.0.11 for k8s 1.28.0+.
```
Comment (palma21) [inline CHANGELOG.md:17]: Component update

Comment (shashankbarsin) [inline CHANGELOG.md]: this is part of the VHDs iirc. can we confirm? if yes, we don't need this line and can remove it

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,34 @@
...
+
+* Bug Fixes 
+  * Fix for some abnormal slow put mc/agentpool operations, caused by hanging connections.
+  * Fix for some throttling issue by increasing secrets store AKV provider cpu limit from 50m to 100m.
+  * Fix for CVE by upgrading Azure file driver version to v1.24.11 on AKS 1.25. [Release notes](https://github.com/kubernetes-sigs/azurefile-csi-driver/releases/tag/v1.24.11).
+  * Fix for Azure CNI Overlay when using Linux Kernel 6.2+ and K8s 1.28+.
+
+  
+* Component Updates
+  * Bumped cloud-controller-manager image to v1.28.2, v1.27.10, v1.26.16 and v1.25.20. [Release notes](https://cloud-provider-azure.sigs.k8s.io/blog/).
```
Comment (shashankbarsin) [inline CHANGELOG.md:26]: can we add a link to each of the versions to point to the respective release from here - https://github.com/kubernetes-sigs/cloud-provider-azure/releases

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,34 @@
...
+
+### Release notes 
+* Features
+  * Added RBAC permissions to CNS for handling swiftv2 CRD resources("multitenantpodnetworkconfigs", "podnetworkinstances" and "podnetworks"). Note that swiftv2 flow is still under development and is not done yet. This change was done to unblock swiftv2 development.
+  * Omsagent addon can now be interacted with using helm commands. The default value is set to true for omsagent addon v2 migration. 
+
+* Bug Fixes 
+  * Fix for some abnormal slow put mc/agentpool operations, caused by hanging connections.
+  * Fix for some throttling issue by increasing secrets store AKV provider cpu limit from 50m to 100m.
+  * Fix for CVE by upgrading Azure file driver version to v1.24.11 on AKS 1.25. [Release notes](https://github.com/kubernetes-sigs/azurefile-csi-driver/releases/tag/v1.24.11).
```
Comment (shashankbarsin) [inline CHANGELOG.md:21]:
```suggestion
  * Fix for CVE by upgrading Azure file driver version to [v1.24.11](https://github.com/kubernetes-sigs/azurefile-csi-driver/releases/tag/v1.24.11) on AKS 1.25.
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,33 @@
...
+* Features
+  * Introduced `acn-multitenancy-editor` ClusterRole to give azure-cns permissions on "multitenantpodnetworkconfigs", "podnetworkinstances", and "podnetworks" resources.
+  * Omsagent addon can now be interacted with using helm commands. The default value is set to true for omsagent addon v2 migration. 
+
+* Bug Fixes 
+  * Fix for some abnormal slow put mc/agentpool operations, caused by hanging connections.
+  * Fix for some throttling issue by increasing secrets store AKV provider cpu limit from 50m to 100m.
+  * Fix for CVE by upgrading Azure file driver version to [v1.24.11](https://github.com/kubernetes-sigs/azurefile-csi-driver/releases/tag/v1.24.11) on AKS 1.25.
+  * Fix for Azure CNI Overlay when using Linux Kernel 6.2+ and K8s 1.28+.
+
```
Comment (qpetraroia) [inline CHANGELOG.md:22]: nit: can we get rid of this extra space ?

File: CHANGELOG.md (hunk 4 above)
Comment (abubinski) [reply CHANGELOG.md]: Talked with engineers and @shashankbarsin and it seems like we don't have links for this

File: CHANGELOG.md (hunk 5 above)
Comment (abubinski) [reply CHANGELOG.md]: Talked to engineer and he confirmed it is a bug fix. I added additional description from him.

File: CHANGELOG.md (hunk 6 above)
Comment (abubinski) [reply CHANGELOG.md]: Fixed, thank you!

File: CHANGELOG.md (hunk 8 above)
Comment (abubinski) [reply CHANGELOG.md:22]: Fixed, thank you!

File: CHANGELOG.md (hunk 5 above)
Comment (abubinski) [reply CHANGELOG.md]: Confirmed it is a bug fix and added detail.

**Review by shashankbarsin — APPROVED** [review]

**Review by qpetraroia — APPROVED** [review]

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,31 @@
...
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* Kubernetes 1.25 is being deprecated at the end of January 2024 and support will transition to our [platform support policy](https://learn.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli#platform-support-policy).
+* No new clusters can be created with [Azure AD Integration (legacy)](https://learn.microsoft.com/azure/aks/azure-ad-integration-cli). Existing AKS clusters with Azure Active Directory integration will keep working. All Azure AD Integration (legacy) AKS clusters will be migrated to [AKS-managed Azure AD](https://learn.microsoft.com/azure/aks/managed-azure-ad) automatically starting from December 1st, 2023. We recommend updating your cluster with AKS-managed Azure AD before December 1st, 2023. This way you can manage the API server downtime during non-business hours.
+
+### Release notes 
+* Features
+  * Introduced `acn-multitenancy-editor` ClusterRole to give azure-cns permissions on "multitenantpodnetworkconfigs", "podnetworkinstances", and "podnetworks" resources.
```
Comment (palma21) [inline CHANGELOG.md:14]: Behavior change

Comment (abubinski) [reply CHANGELOG.md]: thanks, fixed!

**Comment by RitikaLaddha** [comment]
Hi @palma21, Sorry for commenting here. But I needed your input on this [Question](https://github.com/Azure/AKS/issues/1595). Is this still not possible to add metric-resolution parameter in metrics-server??

**Pull request merged**